sdks/go/pkg/beam/core/graph/coder/varint_test.go

```diff
@@ -71,24 +77,26 @@ func TestEncodeDecodeVarInt(t *testing.T) {
 	}
 
 	for _, test := range tests {
-		var buf bytes.Buffer
```

Comment:
Idle thought, but maybe this test could be parallelized? (I just read through the testing docs and noticed testing.T.Parallel)

Reply:
It can be, but it's not necessary, the tests run in less than a second anyway.